### ISSUE TITLE
Unable to Submit

### ISSUE THREAD

**Issue by msceracookie** [freeCodeCamp/freeCodeCamp#15005]
Issue is happening frequently.  Adding correct code (the code below may, or may not be correct, and the icons do not change, even when code is correct.  After several hard refreshes, code will be accepted.  This has happened several times with several pages.  Computer was restarted, as well, and issues remained.

```js
<script>
  $(document).ready(function() {
    $("button").addClass(animated);
    $(".btn").addClass(shake);
    $("#target1").addClass(btn-primary)
    $("#target1").addClass(shake).addClass(animated).addClass(btn-primary);
  });
</script>
```

![screenshot at may 22 11-56-08](https://cloud.githubusercontent.com/assets/27818664/26319738/c72185a2-3ee5-11e7-8a11-c37d6bce54a6.png)

**Comment by texas2010** [comment]
@msceracookie what challenge name?

oh, it seems you forget to use double quote.  that is why it won't pass.

**Comment by msceracookie** [comment]
I have tried this, as well.  Currently, I am not able to view the tasks.  If I clear my code, this will show up, sometimes keeping my code and sometimes removing it.

**Comment by texas2010** [comment]
@msceracookie I am sorry, I can't provide a help to you on here.

it won't pass because your code is incorrect and you forget to use double quote.
Please use the [Help chat room](https://gitter.im/FreeCodeCamp/Help) for getting challenge related help.

GitHub Issues are for reporting bugs on the website only.
Happy Coding.

**Comment by msceracookie** [comment]
Yes, I understand that, however, on several other occasions, on different tasks, my code has been correct and that process still does not proceed.  After refreshing a few times, still while using the same code, the items do resolve and I am able to move forward.  In fact, this item was completed and I was able to move forward.  It wasn't until after viewing the Map that I realized that it wasn't checked off and returned.  This is a bug and not faulty coding.
